front/src/App.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div id="app">
+    <a href="https://github.com/pniedzwiedzinski/get-lucky" class="GitHub">
+      <img src="../assets/GitHub.png" alt="GitHub">
+    </a>
+    <div class="container">
+      <h1>Szczęśliwy Numerek</h1>
+      <div class="loading">
+        <span class="dot dot1"></span>
+        <span class="dot dot2"></span>
+        <span class="dot dot3"></span>
+        <h2>{{number}}</h2>
+      </div>
+      <h4>{{error}}</h4>
+      <div class="curl">
```

Comment:
Lepiej by było to nazwać `.terminal`, `.code` lub coś takiego. Bo w przyszłości planuje dodać więcej przykładów nie koniecznie na curl

Reply:
Zmienione